File: hh-back/src/main/java/gloit/hiperionida/helios/model/ReciboModel.java
package gloit.hiperionida.helios.model;

import gloit.hiperionida.helios.util.model.AbsAuditoriaModel;

import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;

@EqualsAndHashCode(callSuper = false)
@Entity
@Getter
@Setter
@SuperBuilder
@Table(name = "recibo")
public class ReciboModel extends AbsAuditoriaModel {
    private Double monto;
    private LocalDateTime fecha;

    public ReciboModel() {
    }

    public ReciboModel(Long id, Long creadorId, LocalDateTime creada, Long modificadorId, LocalDateTime modificada, Long eliminadorId, LocalDateTime eliminada, Double monto, LocalDateTime fecha) {
        super(id, creadorId, creada, modificadorId, modificada, eliminadorId, eliminada);
        this.monto = monto;
        this.fecha = fecha;
    }
}

File: hh-front/src/models/recibo_model.js
import { UsuarioModel } from './usuario_model'

export class ReciboModel {
  constructor () {
    this.id = ''
    this.monto = ''
    this.fecha = ''

    this.creador = new UsuarioModel()
    this.creada = ''
    this.modificador = new UsuarioModel()
    this.modificada = ''
    this.eliminador = new UsuarioModel()
    this.elimianda = ''
  }
}
